appveyor.yml

```diff
@@ -10,28 +15,36 @@ environment:
   CHROMAPRINT_FPCALC_VERSION: 1.4.2
   DISCID_VERSION: 0.6.1
   matrix:
-  - PYTHON: C:\Python36
-    PYTHON_VERSION: 3.6.1
-    PYTHON_ARCH: 32
-  - PYTHON: C:\Python35
-    PYTHON_VERSION: 3.5.3
-    PYTHON_ARCH: 32
-build: off
-test_script:
+  - PYTHON: C:\Python35-x64
+    python_version: 3.5.3
+    python_arch: 64
+  - PYTHON: C:\Python36-x64
```

Comment:
Does this mean we're now only providing x86_64 builds? AFAICT 1.4.2 was still compatible with 32bit Windows versions. I don't have strong feelings about this, but I'm afraid this will not show up in any changelog when 2.0 is released if there's no jira ticket or something.